src/test/Fixtures/BoxSearch/GetSearchItemsIncludingSharedLinks200.json

```diff
@@ -0,0 +1,86 @@
+{
+  "entries": [
+    {
+      "accessible_via_shared_link": "https://www.box.com/s/vspke7y05sb214wjokpk",
```

Comment:
Assuming we're waiting for the `type` element to be implemented before adding here, right?
I see it's already added to `parseJSONMember` in BoxSearchSharedLink.java.